lib/main.js

```diff
@@ -196,12 +196,8 @@ export default {
         if (version.major > 2) {
           // PHPCS v3 and up support this with STDIN files
           parameters.push(`--ignore=${ignorePatterns.join(',')}`);
-        } else {
-          // We must determine this ourself for lower versions
```

Comment:
I'd like this comment kept in there as without it there is nothing explaining why we are doing this.